src/lxc/attach.c

```diff
@@ -997,6 +1016,8 @@ static int attach_child_main(void* data)
 	int fd;
 	uid_t new_uid;
 	gid_t new_gid;
+	gid_t new_groups[50];
```

Comment:
I just  picked a number that I figured was going to be ample for practically all cases. Could of course go an count how would be needed too, but that seemed like more work than was warranted.